packages/zoe/src/contractSupport/priceAuthorityInitial.js

```diff
@@ -0,0 +1,118 @@
+// @ts-check
+import { E } from '@endo/far';
+import { Far } from '@endo/marshal';
+
+import { makeNotifier } from '@agoric/notifier';
+import { AmountMath } from '@agoric/ertp';
+
+import { multiplyBy } from './ratio.js';
+
+/** @template T @typedef {import('@endo/eventual-send').EOnly<T>} EOnly */
+
+/**
+ * Override `makeQuoteNotifier`, `quoteGiven` to provide an initial price
+ * in case one is not yet available from the source.
+ *
+ * Mainly for testing vaults without waiting for oracle operators to PushPrice.
+ *
+ * @param {Ratio} initialPrice
```

Comment:
"initial" is implied by the transform. consider `priceOutPerIn` or something that would help prevent brand slot mistakes